### PR TITLE
feat: Add enlarged QR Image + Increase brightness

### PR DESCRIPTION
**Pull request by anhanh11001**
Detail:
- Set up click listener to enlarge QR Image
- Increate brightness when clicking on QR Image.

Fixes: #1215

Screenshots for the change:
Somehow my screen recorder doesn't record the change in brightness. But it increases the brightness to maximum when dialog appear and return to current brightness when dismissing the dialog
ezgif-4-ef9e53ad591c

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsFragment.kt [hunk 1]
```diff
@@ -24,6 +25,7 @@ import kotlinx.android.synthetic.main.fragment_order_details.view.orderDetailCoo
 import kotlinx.android.synthetic.main.fragment_order_details.view.orderDetailsRecycler
 import kotlinx.android.synthetic.main.fragment_order_details.view.progressBar
 import kotlinx.android.synthetic.main.item_card_order_details.view.orderDetailCardView
+import kotlinx.android.synthetic.main.item_enlarged_qr.view.*
```
Comment (ShridharGoel) [inline app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsFragment.kt]: Please avoid wildcard imports.

**Comment by ShridharGoel** [comment]
Shouldn't the enlarged QR code be a little bigger than this? Also, in my opinion, the 'Cancel' button can be removed and instead, clicking outside the box should close it.

File: app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsFragment.kt (hunk 1 above)
Comment (anhanh11001) [reply app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsFragment.kt]: yeah, I forgot to check for this

**Comment by anhanh11001** [comment]
> Shouldn't the enlarged QR code be a little bigger than this? Also, in my opinion, the 'Cancel' button can be removed and instead, clicking outside the box should close it.

Ok, I'll try to find way to improve the size. Putting image view into a dialog was a bit trickier than I thought yesterday.

**Pull request merged**